test/end-to-end/cases/focus-vs-pending-attribute.xspec

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	focus-vs-pending-element.xspec copies this file.
+	When modifying this file, check whether focus-vs-pending-element.xspec needs any additions or
+	updates.
+-->
+<x:description query="x-urn:test:do-nothing" query-at="do-nothing.xqm" schematron="do-nothing.sch"
+	stylesheet="do-nothing.xsl" xml:base="../../" xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:scenario label="Expect" shared="yes">
+		<x:expect label="x:expect" test="true()" />
+	</x:scenario>
```

Comment:
In `focus-without-pending.xspec`, I like how the labels in `x:expect` describe what is supposed to happen. I think you're not doing that here because you're trying to exhibit the current behavior without passing judgment on it yet; that will come later. Let me know if I'm misunderstanding.

Reply:
Thanks for reviewing this.
Right. I'm not making judgement here. Actually I still do not exactly know what is supposed to happen. 🙄